src/test/java/dev/vasyl/proj/repository/BookRepositoryTests.java

```diff
@@ -0,0 +1,77 @@
+package dev.vasyl.proj.repository;
+
+import dev.vasyl.proj.config.TestContainersConfig;
+import dev.vasyl.proj.model.Book;
+import dev.vasyl.proj.model.Category;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import java.math.BigDecimal;
+import java.util.Set;
+
+@DataJpaTest
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+public class BookRepositoryTests {
+
+    private static final Book BOOK = new Book();
+    private static final Category CATEGORY = new Category();
```

Comment:
It's not a constants. I am not sure, but maybe you can simply remove these fields
```suggestion
    private static final Book book = new Book();
    private static final Category categor = new Category();
```